### PR TITLE
Remove some unnecessary calls and casts at `StringUtil`, add type and return declarations

### PR DESCRIPTION
**Pull request by phansys**
|Q            |A  |
|---          |---|
|Fixed tickets|n/a|
|License      |MIT|

### CODE REVIEW

File: src/Util/StringUtil.php [hunk 1]
```diff
@@ -18,25 +18,25 @@
  */
 final class StringUtil
 {
-    public static function splitLines($input)
+    public static function splitLines(string $input): string
```
Comment (cordoval) [inline src/Util/StringUtil.php]: is this php 7 syntax?

**Comment by cordoval** [comment]
👍

File: src/Util/StringUtil.php (hunk 1 above)
Comment (phansys) [inline src/Util/StringUtil.php]: Yes @cordoval.

**Comment by phansys** [comment]
I must check the tests, some of them are failing. Should we update the php constraint to `php: ^7.0` at `composer.json`? Or is the tool intended to support older PHP versions for usage outside Docker?

File: src/Util/StringUtil.php (hunk 1 above)
Comment (sstok) [inline src/Util/StringUtil.php]: It returns an array btw 👍 7.1 supports `string[]` IIRC.

Comment (phansys) [inline src/Util/StringUtil.php]: But our test suite only supports (officially) php 7.0.8 currently. I must change to `array` by now.

**Comment by phansys** [comment]
@cordoval: https://github.com/gushphp/gush/blob/master/composer.json#L9

File: src/Util/StringUtil.php (hunk 1 above)
Comment (sstok) [inline src/Util/StringUtil.php]: Just saying ;)

**Comment by cordoval** [comment]
constraint removed

**Pull request merged**